online/src/app/edit-product-form/edit-product-form.component.html

```diff
@@ -0,0 +1,23 @@
+<form [formGroup]="editForm" *ngIf="editForm">
```

Comment:
You basically copy/pasted what you have on add. This is redundant and instead you should use the same component for the form here.